apps/rehabstod/src/pages/CurrentSickLeaves/CurrentSickLeaves.tsx

```diff
@@ -2,7 +2,7 @@ import { IDSContainer } from '@frontend/ids-react-ts'
 import { useEffect } from 'react'
 import { useSelector } from 'react-redux'
 import { Outlet, useNavigate, useParams } from 'react-router-dom'
-import { useGetSickLeavesMutation, useGetUserQuery } from '../../store/api'
+import { useGetUserQuery, useLazyGetSickLeavesQuery } from '../../store/api'
```

Comment:
I get an error on this line
> Module '../../store/api"' has no exported member 'useLazyGetSickLeavesQuery'

Reply:
It's because the summary PR needs to be merged first, they overlap in the api

Reply:
Alright!